### ISSUE TITLE
Add information about compatible versions of node to the README.

### ISSUE THREAD

**Issue by motlin** [twosigma/git-meta#689]
I followed the instructions in [README.md](https://github.com/twosigma/git-meta/blob/master/README.md) and got a compiler error. I tried upgrading node to 11.x and then downgrading to 8.x and then it worked with 8.x.

Assuming git-meta can't be compatible with a wide range of versions of node, I'd suggest adding compatibility info to the README, and maybe a note about nvm.

**Comment by jhedwardyang** [comment]
Yup absolutely. Node 10.x LTS started on 2018/10 do we should do the work to support it. As for non-LTS current releases (eg. 11.x), it seems unlikely we'd support them.

**Comment by Meesujit** [comment]
can you assign it to me.
